### DISCUSSION TITLE
New model replacements folder

### DISCUSSION THREAD

**Discussion by pjinnes** [APSIMInitiative/ApsimX#9141] [Q&A] [answered]
Can someone give me some guidance on the procedure for creating a prototype model (will be based on the Sorghum model). I have added the sorghum model to a Replacements folder and initially modified it there, but after saving and reloading the changes are gone and the model becomes read only.
Have just now had a look at PidgeonPea prototype and can see the .json file has been pasted to the Replacements? and that I'm guessing enables mods to be made

**Comment by hol353** [comment, marked as answer]
To be able to modify a model under replacements you need to grab a portion of the .json resource from the resources directory (https://github.com/APSIMInitiative/ApsimX/tree/master/Models/Resources) and paste onto resources. 

The portion is from the second open curly bracket (always line 7), to the matching close bracket i.e. the second last closing bracket (line 9831 for Sorghum.json).

**Reply by Xiguadedao** [reply]
Hello, excuse me, I've been also running into this issue recently. I downloaded the latest version of ApsimNG (but the one I downloaded last December also had this issue), and it was OK to add Wheat/Maize/Nutrient to the Replacement for the first time and set it up, but when I save and reload, it goes back to the initial state and read-only.

I've read your reply carefully, but don't understand what exactly "paste onto resoures" is? Can you give me some guidance?😭Thank you very much!!

**Reply by pjinnes** [reply]
I think it meant paste onto replacements

**Reply by Xiguadedao** [reply]
It works!! Thank you🙂
